build-tools/automation/guardian/source.gdnsuppress

```diff
@@ -0,0 +1,236 @@
+      "justification": "Reference to an Android logging function.",
+      "createdDate": "2023-02-22 23:55:29Z",
+      "expirationDate": null,
+      "type": null
+    },
+    "8e5400e0233c8d887ad48bd8a48e8a7be5a579f9eefad521419b6df0828bbfac": {
+      "signature": "8e5400e0233c8d887ad48bd8a48e8a7be5a579f9eefad521419b6df0828bbfac",
+      "alternativeSignatures": [],
+      "target": "src/Mono.Android/Android.Util/Log.cs",
+      "memberOf": [
+        "default"
+      ],
+      "tool": "policheck",
+      "ruleId": "80418",
+      "justification": "Reference to an Android logging function.",
+      "createdDate": "2023-02-22 23:55:29Z",
+      "expirationDate": null,
+      "type": null
+    },
+    "06af52be6b6f87455b1db2eb6e631e783f1dacaf607c9b5f34cdee669992c8b5": {
+      "signature": "06af52be6b6f87455b1db2eb6e631e783f1dacaf607c9b5f34cdee669992c8b5",
+      "alternativeSignatures": [],
+      "target": "src/Mono.Android/Android.Util/Log.cs",
+      "memberOf": [
+        "default"
+      ],
+      "tool": "policheck",
+      "ruleId": "80418",
+      "justification": "Reference to an Android logging function.",
+      "createdDate": "2023-02-22 23:55:29Z",
+      "expirationDate": null,
+      "type": null
+    },
+    "a2b4d032c59a9d1211d218c3cd550cf8febb369941d70284d07d03ebee855bc0": {
+      "signature": "a2b4d032c59a9d1211d218c3cd550cf8febb369941d70284d07d03ebee855bc0",
+      "alternativeSignatures": [],
+      "target": "src/monodroid/jni/logger.cc",
+      "memberOf": [
+        "default"
+      ],
+      "tool": "policheck",
+      "ruleId": "79668",
+      "justification": "Reference to find first set bit function.",
+      "createdDate": "2023-02-22 23:55:29Z",
+      "expirationDate": null,
+      "type": null
+    },
+    "1c87b45a6044d205dc3f3562f349c238f7cabe22b4609da762df9dc44151e9fb": {
+      "signature": "1c87b45a6044d205dc3f3562f349c238f7cabe22b4609da762df9dc44151e9fb",
+      "alternativeSignatures": [],
+      "target": "src/monodroid/jni/logger.cc",
+      "memberOf": [
+        "default"
+      ],
+      "tool": "policheck",
+      "ruleId": "79668",
+      "justification": "Reference to find first set bit function.",
+      "createdDate": "2023-02-22 23:55:29Z",
+      "expirationDate": null,
+      "type": null
+    },
+    "a6639098c4785509a4215c9e2fc10f82c06fce461915dc11a00227ddec558845": {
+      "signature": "a6639098c4785509a4215c9e2fc10f82c06fce461915dc11a00227ddec558845",
+      "alternativeSignatures": [],
+      "target": "src/monodroid/jni/logger.cc",
+      "memberOf": [
+        "default"
+      ],
+      "tool": "policheck",
+      "ruleId": "79668",
+      "justification": "Reference to find first set bit function.",
+      "createdDate": "2023-02-22 23:55:29Z",
+      "expirationDate": null,
+      "type": null
+    },
+    "d6b3df0b1d35cb4acec6a954acc145c9ec22041cd463b94ff080682c65a9bd62": {
+      "signature": "d6b3df0b1d35cb4acec6a954acc145c9ec22041cd463b94ff080682c65a9bd62",
```

Comment:
Only concern here -- is random security tooling going to complain about these? They *look* like API keys, where I could see future tools complaining about them.

Reply:
I don't think we'll have to worry about this, if some new scan complains about these we would hopefully only see it in a PR build when changing the underlying template.